marklogic-data-hub-central/ui/src/components/base-entities-facet/base-entities-facet.tsx

```diff
@@ -177,7 +177,7 @@ const BaseEntitiesFacet: React.FC<Props> = (props) => {
           let finalColor = color ? color : "#EEEFF1";
           if (name) {
             return (
-              <HCTooltip text={ExploreGraphViewToolTips.entityToolTip} placement="top" id="baseEntityToolTip">
+              <HCTooltip text={ExploreGraphViewToolTips.entityToolTip} placement="top" id="baseEntityToolTip" key={name}>
```

Comment:
This change fixes a react component warning in the console related to unique keys.